doc/api/fs.md

```diff
@@ -1035,8 +1038,6 @@ changes:
                  whether to perform path resolution for symlinks.
 -->
 
-> Stability: 1 - Experimental
```

Comment:
Other stable `fs` methods doesn't have this line, so I thought it's fine to remove it completely instead of using `Stability: 2 - Stable`.